ssl-config/src/main/java/com/palantir/remoting1/config/ssl/SslSocketFactories.java

```diff
@@ -46,6 +49,11 @@ public static SSLSocketFactory createSslSocketFactory(SslConfiguration config) {
         return sslContext.getSocketFactory();
     }
 
+    public static SSLSocketFactory createSslSocketFactory(Map<String, byte[]> trustCertificatesByAlias) {
```

Comment:
you'll have to explain somewhere what exactly you mean by "trustCertificatesByAlias"